src/test/java/teammates/test/cases/webapi/DowngradeAccountActionTest.java

```diff
@@ -23,7 +27,27 @@ protected String getRequestMethod() {
     @Override
     @Test
     protected void testExecute() {
-        // TODO
+        InstructorAttributes instructor1ofCourse1 = typicalBundle.instructors.get("instructor1OfCourse1");
+
+        loginAsAdmin();
+
+        ______TS("Not enough parameters");
+
+        verifyHttpParameterFailure();
+
+        ______TS("Typical case");
+
+        String[] params = {
+                Const.ParamsNames.INSTRUCTOR_ID, instructor1ofCourse1.getGoogleId(),
+        };
+
+        DowngradeAccountAction a = getAction(params);
+        JsonResult r = getJsonResult(a);
+
+        MessageOutput response = (MessageOutput) r.getOutput();
+
+        assertEquals(HttpStatus.SC_OK, r.getStatusCode());
+        assertEquals("Instructor account is successfully downgraded to student.", response.getMessage());
     }
```

Comment:
Consider adding a failure case for downgrading an invalid account? (non-existent)